lib/ui_components/docu_cop.rb

```diff
@@ -0,0 +1,77 @@
+module UiComponents
+  module DocuCop
```

Comment:
If I understood this correctly, this class does two thing:
- check the documentation
- ensure valid parameters are given at runtime

Shouldn't that be two completely separate classes?